.eslintrc.json

```diff
@@ -18,6 +18,18 @@
         "libraries/botframework-schema/**/*",
         "libraries/streaming-e2e-tests/**"
     ],
+    "overrides": [
+        {
+            "files": [ "*.test.js" ],
```

Comment:
In the interest in one day writing Typescript tests, perhaps use `{js,ts}` as the extension.

Reply:
Done!